openlibrary/components/BulkSearch/components/TableRow.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <tr>
+        <td>{{ index + 1 }}</td>
+        <td>{{ bookMatch.extractedBook.title }}</td>
+        <td>{{ bookMatch.extractedBook.author }}</td>
+        <td>
+            <div class="bookCards">
+                <a :href="searchUrl">L</a>
```

Comment:
```suggestion
                <a :href="searchUrl" title="View results in Open Library">🔎</a>
```

Reply:
Well, that's a placeholder change that I didn't intend  to keep for this long. Changed!